packages/templates/src/raw/travel.json

```diff
@@ -753,7 +768,13 @@
       ],
       "type": "page",
       "id": "3cdf0fde-f515-477d-8688-472c4cc5a8b3",
-      "backgroundColor": { "color": { "r": 255, "g": 255, "b": 255 } },
+      "backgroundColor": {
```

Comment:
Might want to run prettier over this to avoid this change

Reply:
noted for next time.